### PR TITLE
Changed FormField validation to include status

### PR DESCRIPTION
**Pull request by ericsoderberghp**
#### What does this PR do?

Changed FormField validation to include status

#### Where should the reviewer start?

doc.js

#### What testing has been done on this PR?

storybook

#### How should this be manually tested?

storybook

#### Do the grommet docs need to be updated?

automatic

#### Should this PR be mentioned in the release notes?

yes

#### Is this change backwards compatible or is it a breaking change?

backwards compatible

### CODE REVIEW

File: src/js/themes/base.js [hunk 1]
```diff
@@ -554,23 +554,21 @@ export const generate = (baseSpacing = 24, scale = 6) => {
...
-          dark: 'dark-3',
-          light: 'dark-3',
-        },
+        color: 'dark-3',
         margin: {
           start: 'small',
         },
       },
+      info: {
+        color: 'text-xweak',
```
Comment (ShimiSun) [inline src/js/themes/base.js]: :)

File: src/js/components/Form/Form.js [hunk 2]
```diff
@@ -13,12 +13,39 @@ const defaultMessages = {
...
+  }
+  /* eslint-enable no-param-reassign */
+};
 
 const Form = forwardRef(
   (
     {
       children,
       errors: errorsProp = defaultErrors,
+      infos: infosProp = defaultInfos,
```
Comment (ShimiSun) [inline src/js/components/Form/Form.js]: on doc.js this prop is called `info`, and not `infos`.

File: src/js/components/Form/stories/All.js [hunk 3]
```diff
@@ -35,6 +35,11 @@ const Example = () => (
                 if (name && name.length === 1) return 'must be >1 character';
                 return undefined;
               },
+              name => {
+                if (name && name.length === 2)
```
Comment (ShimiSun) [inline src/js/components/Form/stories/All.js]:
```suggestion
                if (name && name.length <= 2)
```

File: src/js/components/Form/Form.js (hunk 2 above)
Comment (ericsoderberghp) [reply src/js/components/Form/Form.js]: Actually, it's `info` for FormField and `infos` for Form. I've added the missing `infos` to Form/doc.js.

**Comment by ShimiSun** [comment]
I think we should add tests for the new props.

**Pull request merged**